src/text/writer.rs

```diff
@@ -76,40 +81,6 @@ impl<W: Write> TextWriter<W> {
         self.output.get_mut()
     }
 
-    /// Causes any buffered data to be written to the underlying io::Write implementation.
```

Comment:
As with the binary writer, most of this file's diff is simply splitting the methods into `impl RawTextWriter {}` and `impl RawWriter for RawTextWriter {}` blocks. There are no logic changes except where I've highlighted them with comments.